tests/utils/test_async_logging_queue.py

```diff
@@ -70,9 +70,10 @@ def test_single_thread_publish_consume_queue(monkeypatch):
         async_logging_queue.flush()
         # 2 batches are sent to the worker thread pool due to grouping, otherwise it would be 5.
         assert mock_worker_threadpool.submit.call_count == 2
-        assert async_logging_queue.is_idle()
+        assert async_logging_queue.is_active()
         assert mock_check_threadpool.shutdown.call_count == 1
         assert mock_worker_threadpool.shutdown.call_count == 1
+        async_logging_queue.shut_down_async_logging()
```

Comment:
Can we use a pytest fixture?

Reply:
Ah yes, seems they all have the same initiation process